### DISCUSSION TITLE
SFSC - SFCC order synchronization

### DISCUSSION THREAD

**Discussion by shayunBBD** [SalesforceCommerceCloud/sfcc-ci#446] [Team Posts]
Hello,

When I try to synchronize an order with order line item from CC to SC by calling continueWithObject(Id), I receive this error:
`FAILED_ACTIVATION, An order must have at least one product`

I don't have any products in my SC, should I synchronize them? Create them manually? Add something to the custom settings? The deployment didn't add any field to the product object, how the code will recognize the right product?

Thank you

**Comment by JoeRida** [comment]
I think the error is related to making an order qctive. This is out of the
box feature, when u try to activate an order u should at least have one
orderitem in the order product related list.
A question to u, did u assign pricebook to the order or not ?

Le mar. 30 juil. 2019 à 08:40, shayunBBD <notifications@github.com> a
écrit :
